.github/workflows/markdown-link-check.config.json

```
@@ -0,0 +1,7 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^http://localhost.*"
```

Shall we also include `https://localhost` for future proof
```suggestion
      "pattern": "^https?://localhost.*"
```